content/en/docs/Examples/services.md

```diff
@@ -526,6 +526,30 @@ service:
       web_url: https://docs.mattermost.com/install/self-managed-changelog.html
 ```
 
+## morpheus65535/bazarr
+Source: https://github.com/morpheus65535/bazarr
+
+- deployed_version - Requires an `API_KEY` which can be retrieved at `Settings/General/Security/API Key`
+```yaml
+service:
+  morpheus65535/bazarr:
+    latest_version:
+      type: github
+      url: morpheus65535/bazarr
+      url_commands:
+      - type: regex
+        regex: v([0-9.]+)$
+    deployed_version:
+      url: https://bazarr.example.io/api/system/status
+      headers:
+      - key: X-API-KEY
+        value: <API_KEY>
```

Comment:
```suggestion
      url_commands:
        - type: regex
          regex: v([0-9.]+)$
    deployed_version:
      url: https://bazarr.example.io/api/system/status
      headers:
        - key: X-API-KEY
          value: <API_KEY>
```
Have indented list items like this on the others

Reply:
Sure, I'll add another commit to indent my previous changes the same way.

Reply:
Oh hadn't noticed I'd missed it on those XD. Thank you :)